torch/distributed/fsdp_common.py

```diff
@@ -0,0 +1,13 @@
+# This file contains stuff that is shared between FSDP-1 and FSDP-2.
+
+import os  # noqa: C101
+
+from typing import Any
+
+from torch.distributed.fsdp import FullyShardedDataParallel as FSDP1
+from torch.distributed._composable.fsdp.fully_shard import FSDPModule as FSDP2
+
+use_fsdp2 = os.environ.get("FSDP_VERSION", "1") == "2"
```

Comment:
nit: should we warn if it's neither 1 or 2?

Reply:
It would be better to be a function `fsdp_version()`.